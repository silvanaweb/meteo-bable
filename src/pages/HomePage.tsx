import React from "react";
import { Header } from "../components/Header/Header.component";
import { CurrentWeather } from "../components/CurrentWeather/CurrentWeather.component";

const HomePage: React.FC = () => {
  return (
    <>
      <Header />

      <div className="pt--xl pb--xl">
        <CurrentWeather />
      </div>
    </>
  );
};

export { HomePage };
