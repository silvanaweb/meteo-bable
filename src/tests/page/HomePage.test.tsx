import { shallow, ShallowWrapper } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";
import { HomePage } from "../../pages/HomePage";

let wrapper: ShallowWrapper;

beforeEach(() => {
  wrapper = shallow(<HomePage />);
});

test("should render HomePage correctly", () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});
