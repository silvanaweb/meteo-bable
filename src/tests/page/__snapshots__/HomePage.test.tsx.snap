// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage correctly 1`] = `
<Fragment>
  <Header />
  <div
    className="pt--xl pb--xl"
  >
    <CurrentWeather />
  </div>
</Fragment>
`;
